src/topology/instances/add_circle.lean

```diff
@@ -280,6 +285,61 @@ begin
     simpa [zero_add] using (equiv_Ico p 0 u).2.2, },
 end
 
+lemma add_order_of_eq_pos_iff {u : add_circle p} {n : ℕ} (h : 0 < n) :
+  add_order_of u = n ↔ ∃ m < n, gcd m n = 1 ∧ ↑(↑m / ↑n * p) = u :=
+begin
+  refine ⟨λ hu, _, _⟩,
+  { rw ← hu at h,
+    obtain ⟨m, h₀, h₁, h₂⟩ := exists_gcd_eq_one_of_is_of_fin_add_order (add_order_of_pos_iff.mp h),
+    refine ⟨m, _, _, _⟩;
+    rwa ← hu, },
+  { rintros ⟨m, h₀, h₁, rfl⟩,
+    exact add_order_of_div_of_gcd_eq_one h h₁, },
+end
+
+variables (p)
+
+/-- The natural bijection between points of order `n` and natural numbers less than and coprime to
+`n`. -/
```

Comment:
Can you be a little bit more specific in the docstring, saying thtat the order `n` points are the points of the form `m/n * p`?

Reply:
Sure, done!